### PR TITLE
Firefox suggest: outcomes & new config

### PR DESCRIPTION
**Pull request by rebecca-burwei**
- Moves Firefox Suggest metrics into outcomes
- Updates existing firefox-suggest-history-vs-offline.toml config to use outcomes
- Adds a new config that also uses outcomes

@scholtzan , I followed the docs [here](https://experimenter.info/jetstream/outcomes) to add outcomes, but please let me know if I need to do anything else to "import" the outcomes into my 2 configs.

@jaredlockhart ^ another opportunity for documentation

### CODE REVIEW

File: smarter-suggestions-in-the-awesomebar.toml [hunk 1]
```diff
@@ -1,5 +1,38 @@
...
+data_source = "clients_daily_custom_windows"
+
+[segments.data_sources.clients_daily_custom_windows]
+from_expression = "mozdata.telemetry.clients_daily"
+window_start = -7
+window_end = 0
 
+[metrics]
 weekly = [
 	"urlbar_search_count",
```
Comment (scholtzan) [inline smarter-suggestions-in-the-awesomebar.toml:37]: You don't have to list the outcome metrics here.
Once the outcome has been merged, you can select it in Experimenter and then all the metrics that are defined in the outcome config will be computed automatically.

**Review by scholtzan — APPROVED** [review]
Looks, good. One thing to change is to remove the metrics from the experiment configs that are now part of the outcome. I left and inline comment with some more explanation.

File: smarter-suggestions-in-the-awesomebar.toml (hunk 1 above)
Comment (rebecca-burwei) [reply smarter-suggestions-in-the-awesomebar.toml:37]: I see, by Experimenter, do you mean there will be things to choose or click on here: https://experimenter.services.mozilla.com/nimbus/firefox-suggest-history-vs-offline

File: outcomes/firefox_desktop/firefox_suggest.toml [hunk 2]
```diff
@@ -0,0 +1,70 @@
+[metrics.urlbar_search_count]
```
Comment (scholtzan) [inline outcomes/firefox_desktop/firefox_suggest.toml:1]: whoops, I missed this earlier. You'll need to have a `friendly_name` and a description that Experimenter will show.
```suggestion
friendly_name = "Firefox Suggest"
description = "Usage and engagement metrics for Firefox Suggest."

[metrics.urlbar_search_count]
```

File: smarter-suggestions-in-the-awesomebar.toml (hunk 1 above)
Comment (scholtzan) [reply smarter-suggestions-in-the-awesomebar.toml:37]: When creating a new experiment you can select Primary and Secondary outcomes in the Metrics tab:
Screen Shot 2021-08-19 at 15 37 11

Once this PR gets merged, "Firefox Suggest" will appear in that dropdown list. When selected and the experiment has been launched, the metrics defined in the outcome will automatically be computed.

Now that these experiments have already been launched, someone from the Experimenter team wll need to update the experiment definitions manually and add this outcome to it (maybe @jaredlockhart can do that?).

There is also some documentation around outcomes here that might be helpful: https://experimenter.info/jetstream/outcomes/

Comment (jaredlockhart) [reply smarter-suggestions-in-the-awesomebar.toml:37]: Yep can do, generally we don't want to alter experiments after they've launched but we can make an exception in this case. @rebecca-burwei Just ping me with the links to the experiments you want to change and which outcomes you want to add in #experimenter-dev and I can add them 👍 Just keep in mind that generally altering an experiment after it launches should be avoided as much as possible.

Comment (rebecca-burwei) [reply smarter-suggestions-in-the-awesomebar.toml]: Thanks @scholtzan and @jaredlockhart ! Will do

**Pull request merged**